### ISSUE TITLE
ACF Wysiwyg Editor fields takes repeated Shortcake values

### ISSUE THREAD

**Issue by StaggerLeee** [wp-shortcake/shortcake#645]
Will try to explain it best I can.
- It is some problem between ACF and Shortcode UI.
- ACF is PRO version, latest.
- Shortcode is updated from Github.
- One ACF Group.
- Several ACF fields attached to Posts, among them two ACF Wysiwyg Editor fields.
- Editing old Shortcake shortcode doesnt change anything in original shortcode. But it adds (for each editing one new) shortcodes in ACF editor field. (I checked by writing numbers for each shortcode updating, 1111, 2222, 3333. All of them ends in ACF editor field, but original is not changed,
- Browser is Chrome.
- I noticed if I keep clicking and updating longer time it pass and actually update shortcode text, but very seldom and unpredictable. Saying this just if it can help troubleshooting.
- Deactivate ACF plugin and Shortcake shortcode works well.

---

Here is my custom code responsible for Shortcake shortcode content. Non of those names are equal to ACF names.

```
add_shortcode( 'article_blocks', function( $attr, $content = '' ) {
        $attr = wp_parse_args( $attr, array(
            'block_text' => '',
        ) );
        ob_start();
        ?>

        <?php if ( ! empty( $attr['block_align'] ) ) : ?>
            <div class='container-block'>
            <?php endif; ?>
        <div class="alert alert-<?php echo esc_html( $attr['color'] ); ?> text-<?php echo esc_html( $attr['text_align'] ); ?> <?php echo esc_html( $attr['block_align'] ); ?>" role="alert">
        <h5><?php echo esc_html( $attr['block_title'] ); ?></h5>
                <?php /*?><?php echo esc_html( $attr['block_text'] ); ?><?php */?>
                <?php echo urldecode( $attr['block_text'] ); ?>
        </div>
        <?php if ( ! empty( $attr['block_align'] ) ) : ?>
               </div>
            <?php endif; ?>

        <?php

        return ob_get_clean();
    } );
```

**Comment by StaggerLeee** [comment]
It is same behaviour as here:
https://github.com/wp-shortcake/shortcake/issues/584

This time it makes a copy of shortcode in ACF editor field on each update.
- No JS errors in console.
- Same problem in Chrome and Firefox.
- Deactivated all plugins except ACF, to exclude them.
- Tried with default theme.

**Comment by StaggerLeee** [comment]
Seems as this is solved with latest 0.7 version. Will keep an eye on it and report back if it happens again.
